packages/components/psammead-section-label/src/index.jsx

```diff
@@ -44,15 +41,13 @@ const SectionLabelWrapper = styled.div`
   ${({ visuallyHidden }) =>
     visuallyHidden &&
     css`
-      // Hide when under 600px
-      @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-        clip-path: inset(100%);
-        clip: rect(1px, 1px, 1px, 1px);
-        height: 1px;
-        overflow: hidden;
-        position: absolute;
-        width: 1px;
-      }
+      // Hide for all breakpoints for top story
```

Comment:
```suggestion
      // Hide for all breakpoints
```
Doesnt need to be top story, just `visuallyHidden` is set :)